source/core/chat_bot_template.py

```diff
@@ -17,14 +17,31 @@
 
 logger = logging.getLogger(__name__)
 
+MAIN, GRAPH = range(2)
 
+chats_status_dict = {}
 # Define a few command handlers. These usually take the two arguments update and
 # context. Error handlers also receive the raised TelegramError object in error.
```

Comment:
Maybe remove this comment? I don't understand what it references to

Reply:
I agree with it.
```suggestion
```